talent/job-descriptions/company-section.md

```diff
@@ -1,6 +1,3 @@
 ## Company 
 
-**Location:** Madrid / Fully remote
-**Seniority:** We are open to all levels of seniority, and adjust compensation accordingly
-
-At source{d} we are building the stack for the next generation of Machine Learning powered developer tools. We have raised over eight million USD so far, and we are currently growing our team. Our current monetization strategy is based on deploying our stack and tooling within large tech companies.
\ No newline at end of file
+At source{d} we are building the technology stack for the next generation of machine learning powered developer tools. We have raised over eight million USD so far, and we are currently growing our team.
```

Comment:
I would mention the open core principles with a link to https://en.wikipedia.org/wiki/Open_core